v0/destinations/appcues/data/AppcuesPageProfile.json

```diff
@@ -0,0 +1,8 @@
+{
+    "userId": "userId",
+    "_lastBrowserLanguage": "context.locale",
+    "_updatedAt": "sentAt",
+    "_userAgent": "context.userAgent",
+    "_currentPageTitle": "context.page.title",
+    "_currentPageUrl": "context.page.url"
```

Comment:
multiple source keys can be there - context.page.url, properties.url. same for title.

Reply:
Yes, will handle them too